xprof/xprof.rb.in

```diff
@@ -883,20 +906,23 @@ def all_env_tracers
 
     # Find and Save the original lib path
     libenv = "LTTNG_UST_#{name.upcase}_#{lib.upcase}"
+    libso = "#{lib}.so"
     if (e = env_fetch_first(libenv))
       LOGGER.warn("#{libenv} was already set, will use this path #{e} for the #{name} loader")
       # TODO: Verify that this guy exist?
-    elsif (libpath = whichlib64("#{lib}.so"))
+    elsif (libpath = whichlib64(libso))
+      h[libenv] = libpath
+    elsif (libpath = find_lib_path(libso, MPITopo.rank_id == 0))
```

Comment:
```suggestion
    elsif (libpath = find_lib_path(libso, MPITopo.master?))
```
@TApplencourt do you want to also check some kind of log level before printing stuff?